### ISSUE TITLE
CDK Overlay - Position Strategy with zoom css on parent

### ISSUE THREAD

**Issue by jgodi** [angular/material2#10924]
#### Bug, feature request, or proposal:
If you are using the `zoom` CSS property to scale down UI elements the positioning of the CDK Overlay are calculated incorrectly.

#### What is the expected behavior?
Overlay positions properly as the non-zoomed styled `div`

#### What is the current behavior?
Overlay positions are calculated incorrectly when the `zoom: 0.9` css property is on the parent container.

#### What are the steps to reproduce?
https://stackblitz.com/edit/cdk-overlay-zoom

#### What is the use-case or motivation for changing an existing behavior?
We have a Chrome only application that we use the `zoom` css property to scale down the UI. Using other methods of scaling does not work, so this is what we have to do.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular - 5.2.10
CDK - 5.2.5

**Comment by pfeigl** [comment]
@jgodi For a quick fix, you can do the following in your trigger directive: `    this._overlayRef.hostElement.style.zoom = "0.9"`

However this brings me the question what the expected behavior actually is. Should the overlay only position correctly or should it also be zoomed accordingly when it is within the zoomed parent?

**Comment by Harikumar-Shankar** [comment]
@pfeigl  I face a similar problem but the solution I need is to position the overlay correctly as original. I am using mat-select and the mat-options are displayed incorrectly after zoom. As we take this example setting zoom to overlayref reduces size but position is incorrect.

**Comment by baljinderkaur** [comment]
Yes I had the same issue. I want to use zoom property in different devices but the mat-select dropdown is not taking right position. Should be solved. But mat-autocomplete is working fine that do not have transform property and animation. Can't we use the same?

**Comment by ThisIsArnab** [comment]
It's angular material ver 10 now and this issue is still open!

**Comment by kimuradev** [comment]
Any news?

**Comment by djok1** [comment]
Using @angular/cdk": "12.2.13" still seems to be a issue any known workarounds?

**Comment by Fomicha** [comment]
Any updates?

**Comment by louisljw** [comment]
Need help...

**Comment by CraigA-ZA** [comment]
Also being impacted by this at the moment.

**Comment by lucianconst** [comment]
Same problem today

**Comment by joewIST** [comment]
This is also an issue for us. An area with css zoom applied doesn't work with: OverlayRef, MatTooltip, CdkDragDrop...

**Comment by BGSilviu** [comment]
Use ::ng-deep in your component.css to override the default styles.
::ng-deep .cdk-overlay-container {
    /* Do you changes here */

}

**Comment by StevenSwitten** [comment]
any update or potential workaround? Scaling the overlay container down doesn't fix the positioning issue

**Comment by Khumozin** [comment]
Any update on this?

**Comment by t0m-4** [comment]
any update ?

**Comment by ptandler** [comment]
we also experienced this issue

**Comment by abdurrahmanyildirim** [comment]
Any progress?

**Comment by Khumozin** [comment]
Any updates?

**Comment by mbilotta** [comment]
I've developed a partial workaround here:

https://gist.github.com/mbilotta/5c64e05d5c0b6f644dcbd2f0fb86d218

https://codesandbox.io/p/devbox/vigorous-https-j89tzx

Please consider that this workaround has some limitations:

- It is only working for CDK versions up to 19.x
- It assumes zoom can only be applied to the root element.

Here there is also an in-depth discussion: https://medium.com/@m.bilotta85/the-zoom-css-property-and-broken-overlays-ebf99723b0db

**Comment by shinkhouse** [comment]
@crisbeto  I want to mention that this may be fixed in Safari 26.4, see notes on Zoom features here:
https://webkit.org/blog/17862/webkit-features-for-safari-26-4/
